packages/twenty-server/src/modules/company/standard-objects/company.workspace-entity.ts

```diff
@@ -273,4 +284,21 @@ export class CompanyWorkspaceEntity extends BaseWorkspaceEntity {
   @WorkspaceIsDeprecated()
   @WorkspaceIsNullable()
   addressOld: string;
+
+  @WorkspaceField({
+    standardId: COMPANY_STANDARD_FIELD_IDS.searchVector,
+    type: FieldMetadataType.TS_VECTOR,
+    label: SEARCH_VECTOR_FIELD_LABEL,
+    description: SEARCH_VECTOR_FIELD_DESCRIPTION,
+    icon: 'IconUser',
+    generatedType: 'STORED',
+    asExpression: getTsVectorColumnExpressionFromFields([
+      { name: NAME_FIELD_NAME, type: FieldMetadataType.TEXT },
```

Comment:
Should we use standardFieldId instead here?

Reply:
In the util at this stage we cannot fetch fieldMetadata based on the standardFieldId, can we?

Reply:
Probably not yet. Should be part of the refactoring of sync-metadata but we already started to use hashmap in some places with standard-id, nameSingular, namePlural, etc... as keys.
You can keep it as it is for now indeed